Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -342,7 +342,10 @@ class BrowserViewController: UIViewController {
 
             addChildViewController(homePanelController!)
         }
-
+        
+        //updates previous home panel
```

Comment:
You can probably drop this comment (I think the code below is pretty self-descriptive).